packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -508,29 +505,28 @@ class SyncWatcher extends BaseWatcher {
 
   async handleTransferRootBondedEvent (event: TransferRootBondedEvent) {
     const { transactionHash, blockNumber } = event
-    const { root, amount } = event.args
-    const logger = this.logger.create({ root })
+    const { root: transferRootHash, amount } = event.args
+    const transferRootId = this.bridge.getTransferRootId(
+      transferRootHash,
+      amount
+    )
+
+    const logger = this.logger.create({ root: transferRootId })
     logger.debug('handling TransferRootBonded event')
 
     try {
-      const bondTransferRootId = this.bridge.getTransferRootId(
-        root,
-        amount
-      )
-
-      logger.debug(`transferRootHash from event: ${root}`)
+      logger.debug(`transferRootHash from event: ${transferRootHash}`)
       logger.debug(`event transactionHash: ${transactionHash}`)
       logger.debug(`event blockNumber: ${blockNumber}`)
       logger.debug(`bondAmount: ${this.bridge.formatUnits(amount)}`)
-      logger.debug(`bondTransferRootId: ${bondTransferRootId}`)
+      logger.debug(`transferRootId: ${transferRootId}`)
 
-      await this.db.transferRoots.update(root, {
-        transferRootHash: root,
+      await this.db.transferRoots.update(transferRootId, {
```

Comment:
The key name is [added as property](https://github.com/hop-protocol/hop/blob/12f3b0d61a843e8e8eb07335a13cde0f826929ca/packages/hop-node/src/db/TransferRootsDb.ts#L248) on `.update()` so in this case it will also write the `transferRootId` to the entry